netpyne_ui/mod_utils.py

```diff
@@ -0,0 +1,42 @@
+import subprocess
+import os
+
+import neuron
+from neuron import h
+
+
+def is_loaded_mechanisms():
+    # copied from:
+    # https://www.neuron.yale.edu/neuron/static/py_doc/modelspec/programmatic/mechtype.html
+    mt = h.MechanismType(0)
+    mname = h.ref('')
+    mnames = list()
+    for i in range(mt.count()):
+        mt.select(i)
+        mt.selected(mname)
+        mnames.append(mname[0])
+
+    # note this check only works for the original hnn model
+    # need to generalize for any model
+
+    if 'hh2' not in mnames:
+        return False
+    else:
+        return True
+
+
+def compileModMechFiles(compileMod, modFolder):
+    # Create Symbolic link
+    if compileMod:
+        modPath = os.path.join(str(modFolder), "x86_64")
+
+        subprocess.call(["rm", "-r", modPath])
```

Comment:
not sure if that's a case but this might throw an Exception when modPath is not present

Reply:
I only moved that code. You're right but probably subprocess fails without throwing an error